src/templates/eye_upper_slim.csv

```diff
@@ -0,0 +1,20 @@
+ID,subset,label
+ID,AI oboInOwl:inSubset,
+CL:0000210,http://purl.obolibrary.org/obo/cl#eye_upper_slim,photoreceptor cell
+CL:0000636,http://purl.obolibrary.org/obo/cl#eye_upper_slim,Mueller cell
+CL:0000748,http://purl.obolibrary.org/obo/cl#eye_upper_slim,retinal bipolar neuron
+CL:0000745,http://purl.obolibrary.org/obo/cl#eye_upper_slim,retina horizontal cell
+CL:0000740,http://purl.obolibrary.org/obo/cl#eye_upper_slim,retinal ganglion cell
+CL:0000561,http://purl.obolibrary.org/obo/cl#eye_upper_slim,amacrine cell
+CL:0000575,http://purl.obolibrary.org/obo/cl#eye_upper_slim,corneal epithelial cell
+CL:0000132,http://purl.obolibrary.org/obo/cl#eye_upper_slim,corneal endothelial cell
+CL:0000306,http://purl.obolibrary.org/obo/cl#eye_upper_slim,crystallin accumulating cell
+CL:0000347,http://purl.obolibrary.org/obo/cl#eye_upper_slim,scleral cell
+CL:0002565,http://purl.obolibrary.org/obo/cl#eye_upper_slim,iris pigment epithelial cell
+CL:0002585,http://purl.obolibrary.org/obo/cl#eye_upper_slim,retinal blood vessel endothelial cell
+CL:0000348,http://purl.obolibrary.org/obo/cl#eye_upper_slim,choroidal cell of the eye
+CL:0009001,http://purl.obolibrary.org/obo/cl#eye_upper_slim,compound eye retinal cell

```

Comment:
Text def for this term: "Any cell in the compound eye, a light sensing organ composed of ommatidia."

This seems to be a very general class that is not populated (no inferred or asserted ancestors). Unsure how much value it will bring to the slim.

Reply:
I agree, I initially added this term when i was using more general terms (eg retinal cell), but due the new criterium, even if it was populated, it shouldn't be in the slim.